src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +55,12 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    @Override
+    public String getPrintableString(Printable... printables) {
+        return null;
```

Comment:
Calling `myAddress.getPrintableString()` should return some kind of string that can be used to display the address values, instead of a null. Additionally, you wouldn't need to pass printable objects as arguments to get the string return value, i.e. `myAddress.getPrintableString(myAddress)` is not needed.